src/Sulu/Bundle/WebsiteBundle/Tests/Functional/Routing/PortalLoaderTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Tests\Functional\Routing;
+
+use Sulu\Bundle\TestBundle\Testing\WebsiteTestCase;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+
+class PortalLoaderTest extends WebsiteTestCase
+{
+    /**
+     * @var KernelBrowser
+     */
+    private $client;
+
+    public function setUp(): void
+    {
+        $this->client = static::createWebsiteClient();
+    }
+
+    public function testPortalRoute()
```

Comment:
unfortunately, the behaviour for `http://example.lo/valid-english-prefix/portal-route` is still wrong. but i dont see an easy way to fix that 😕